common/decisions/wc_cen_kal_decisions.txt

```diff
@@ -0,0 +1,370 @@
+			}
+			set_global_variable = {
+				name = flag_created_quilboar_empire
+				value = scope:founder
+			}
+		}
+	}
+
+	ai_potential = {
+		always = yes
+	}
+
+	ai_will_do = {
+		base = 100
+	}
+}
+
+take_the_walk_quilboar = {
+	ai_check_interval = 12
+	picture = "gfx/interface/illustrations/decisions/quilboar_take_walk.dds"
+	desc = take_the_walk_quilboar_desc
+	title = take_the_walk_quilboar_name
+	selection_tooltip = take_the_walk_quilboar_tt
+	confirm_text = take_the_walk_quilboar_confirm
+	major = yes
+
+	cooldown = { years = 2 }
+
+	is_shown = {
+		OR = {
+			faith = faith:agamaggan_worship
+			culture = culture:quilboar
+		}
+		OR = {
+			has_trait = infirm
+			has_trait = weak
+			has_trait = eunuch
+			has_trait = excommunicated
+			has_trait = sickly
+			has_trait = impotent
+			has_trait = incapable
+			has_trait = blind
+			has_trait = one_legged
+			age > 60
+		}
+		is_imprisoned = no
+	}
+	
+	cost = { piety = 25 }
+	
+	effect = { trigger_event = cen_kal.1062 }
+}
+
+#Create Centaur Empire
+create_centaur_empire = {
+	ai_check_interval = 12
+	picture = "gfx/interface/illustrations/decisions/centaur_empire.dds"
+	desc = create_centaur_empire_desc
+	title = create_centaur_empire_name
+	selection_tooltip = create_centaur_empire_tt
+	confirm_text = create_centaur_empire_confirm
+	major = yes
+
+	ai_check_interval = 0
+
+	is_shown = {
+		is_landed = yes
+		highest_held_title_tier <= tier_kingdom
+		OR = {
+			faith = faith:theradras_worship
+			culture = culture:centaur
+		}
+		NOT = {  #Can only do it once.
+			is_target_in_global_variable_list = {
+				name = unavailable_unique_decisions
+				target = flag:flag_created_centaur_empire
+			}
+		}
+		root.primary_title = { is_mercenary_company = no }
+	}
+
+	is_valid = {
+		custom_description = {
+			text = found_empire_decision_two_kingdoms_held
+			any_held_title = {
+				count >= 2
+				tier = tier_kingdom
+			}
+		}
+		title:c_maraudon = {
+			OR = {
+				holder = root
+				custom_tooltip = { text = found_empire_location_owned_or_vassal_tt holder = { is_vassal_or_below_of = root } }
+			}
+		}
+	}
+
+	is_valid_showing_failures_only = {
+		is_available_adult = yes
+		is_imprisoned = no
+		is_independent_ruler = yes
+	}
+
+	cost = {
+		prestige = 2000
+		piety = 1000
+	}
+
+	effect = {
+		save_scope_as = founder
+		create_new_custom_empire_decision_effect = { NAME = centaur_empire_name } 
+		add_prestige_level = 1
+		root.primary_title = { set_color_from_title = title:k_wildhammer }
+		
+		trigger_event = cen_kal.1065
+		#Notify other players.
+		every_player = {
+			limit = {
+				NOT = { this = root }
+				is_within_diplo_range = { CHARACTER = root }
+			}
+			trigger_event = cen_kal.1066
+		}
+		hidden_effect = {
+			add_to_global_variable_list = {
+				name = unavailable_unique_decisions
+				target = flag:flag_created_centaur_empire
+			}
+			set_global_variable = {
+				name = flag_created_centaur_empire
+				value = scope:founder
+			}
```

Comment:
```suggestion
```
Never used, delete it

Reply:
It's used in the decision to check that the character hasn't made it before, nor has any other character made that distinct empire.